core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -164,6 +169,16 @@ private[history] class FsHistoryProvider(conf: SparkConf, clock: Clock)
     }
   }
 
+  /**
```

Comment:
Similar to what I commented on before, that kind of doc is not super helpful, since that is already explained in the interface's doc.